cl/search/models.py

```diff
@@ -2731,6 +2731,11 @@ class OpinionCluster(AbstractDateTimeModel):
         storage=IncrementingAWSMediaStorage(),
         blank=True,
     )
+    harvard_id = models.CharField(
+        help_text="The ID of the item in the Caselaw Access Project (Harvard)",
+        default=0,
+        db_index=True,
```

Comment:
`db_index` is sort of soft-deprecated these days. Those in the know say not to use it, and to prefer using the `indexes` meta variable. If you do that, it's worth seeing if you can also remove the `varchar_pattern_ops` index that you can see in the SQL, since we don't need it.

Reply:
SGTM. Very unfair to the DB to be using string for the numeric identifier – indexing will much improved by the switch to integer.

I see other bits of `OpinionCluster` using `db_index`; I can see switching those over could be a worthwhile, albeit separate, bit of housekeeping. Let me know.

Reply:
It's sort of on my list of things to do. I think the trick is to just add new indexes and then delete the old ones, but you could also get clever and figure out the names of the old indexes so that the actual DB migration isn't needed. And then, of course, when you get to this sort of a place, it sort of also feels like maybe it's just fine to leave alone.